packages/nav/index.demo.tsx

```diff
@@ -1,7 +1,7 @@
 import { h, Component } from 'skatejs';
 import { Nav } from './Nav';
-import { NavItem } from './Nav-item';
-import { NavContent } from './Nav-content';
+import { NavItem } from './NavItem';
+import { NavContent } from './NavContent';
```

Comment:
All the 3 imports should be from **index** now, as you have added it there ;-)

So:
```
import { Nav } from './index';
import { NavItem } from './index';
import { NavContent } from './index';
```